package.json

```diff
@@ -10,11 +10,13 @@
   },
   "homepage": "https://github.com/Fitbit/fitbit-sdk-toolchain#readme",
   "scripts": {
-    "build": "rm -rf lib && tsc -p tsconfig.build.json",
+    "build": "rm -rf lib && npm run lint && tsc -p tsconfig.build.json",
```

Comment:
CI runs `yarn lint` and `yarn test`. There is no need to run the `lint` script as part of `build` too, since that will interfere with local development _without_ having any effect on whether CI will block PRs.

Reply:
Makes sense :)